src/app/collective-rewards/rewards/MyRewards.tsx

```diff
@@ -42,6 +41,9 @@ export const Rewards: FC<{ builder: Address; gauge: Address }> = ({ builder, gau
     >
       <RewardsSection>
         <RewardsSectionHeader
+          onSettings={() => {
+            router.push('/user/settings?type=builder')
```

Comment:
I'm not sure if we need to specify the different type here. IIRC we don't have other settings ATM

Reply:
we will have backers settings soon though won't we